src/main/java/com/iota/iri/service/snapshot/impl/SnapshotProviderImpl.java

```diff
@@ -4,13 +4,7 @@
 import com.iota.iri.conf.SnapshotConfig;
 import com.iota.iri.model.Hash;
 import com.iota.iri.model.HashFactory;
-import com.iota.iri.service.snapshot.Snapshot;
-import com.iota.iri.service.snapshot.SnapshotException;
-import com.iota.iri.service.snapshot.SnapshotMetaData;
-import com.iota.iri.service.snapshot.SnapshotProvider;
-import com.iota.iri.service.snapshot.SnapshotState;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.iota.iri.service.snapshot.*;
```

Comment:
This is an automatic import optimization that is specified by the guideline. Since we have more than 5 elements a wildcard is allowed:
https://github.com/iotaledger/iri/blob/dev/STYLEGUIDE.md#avoid-wildcard-imports